docker/jdk9/x86_64/ubuntu/Dockerfile-openj9

```diff
@@ -0,0 +1,39 @@
+# Copyright IBM Corporation 2017.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk_container
+
+MAINTAINER Mark Stoodley <mstoodle@ca.ibm.com>
+
+# Install required OS tools
+RUN apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    autoconf \
```

Comment:
`autoconf` will be in the base image.  I don't think it needs to be explicitly installed here.

Reply:
I don't believe that's true. At least, when I removed this line, openj9 failed to build in a fairly catastrophic way. Note that autoconf isn't needed to run the configure scripts; it's needed to generate such a script.